test/multiple-transactions-test.coffee

```diff
@@ -0,0 +1,89 @@
+{assert} = require('chai')
+drafter = require('drafter')
+
+metamorphoses = require('../src/metamorphoses')
+
+
+source = '''
+  FORMAT: 1A
+
+  # Test API
+  ## Test [/test]
+  ### Test Test Test [GET]
+
+  + Request
+  + Response 200 (application/json)
+      + Body
+
+              {"message": "Hello World!"}
+
+  + Request User Error
+  + Response 400 (application/json)
+      + Body
+
+              {"error": "This is error :("}
+
+  + Request Something Not Found
+  + Response 404 (application/json)
+      + Body
+
+              {"error": "This is error :("}
+'''
+
+
+isApiElement = (element) ->
+  return false if element.element isnt 'category'
+  return element.meta?.classes.indexOf('api') isnt -1
+
+
+describe.only('Multiple Transactions', ->
```

Comment:
@honzajavorek was the `.only` a mistake?

This disables all the other tests (which I might add are not passing in master).

Reply:
Yes, mistake 😱